data/tmpl/show_data.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Make Firefox your own</title>
+    <link rel="stylesheet" href="/data/css/all-aboard.css" media="screen" />
+  </head>
+  <body>
+    <section id="import" class="sidebar-content">
+      <header>
+        <img src="/data/media/moving-truck.png" width="65" height="40" class="moving-truck" alt="Image of moving truck" />
+      </header>
+      <main>
+        <h1>We've got your back.</h1>
+        <p>Firefox has already imported your preferences from other browsers you've used in the past.</p>
+        <h2>History</h2>
+        <div class="user-data">
+          <p id="history1"></p>
+          <p id="history2"></p>
+        </div>
+        <h2>Bookmarks</h2>
+        <div class="user-data">
+          <p id="bookmarks1"></p>
+          <p id="bookmarks2"></p>
+        </div>
+        <h2>Passwords</h2>
+      </main>
+      <footer>
+        <a href="about:home">Get started</a>
+        <br>
+        <a href="about:home">Remove Imported Data</a>
```

Comment:
Have we decided on the style of these? Should they be styled as buttons? Should one by a simple text link and the other a button? I am thinking have 'Remove imported data' first as a button (actual button element) and then the 'Get started' as a plain text link. Thoughts @chrismore and @FrancescoSTL